tests/ChatTests/Groups.hs

```diff
@@ -1226,7 +1226,7 @@ testGroupMessageDelete =
   testChat3 aliceProfile bobProfile cathProfile $
     \alice bob cath -> do
       createGroup3 "team" alice bob cath
-      disableFullDeletion3 "team" alice bob cath
+      -- disableFullDeletion3 "team" alice bob cath
```

Comment:
is it commented because role is set to Nothing now? maybe remove altogether from tests then?

Reply:
it has to be set back to moderators only, but it was rushed - the role has to roll out first.